debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCObjectHeapIteratorAddressOrderedList_V1.java

```diff
@@ -133,12 +140,37 @@ public int compare(U8Pointer[] o1, U8Pointer[] o2)
 					return o1[0].compare(o2[0]);
 				}
 			});
-		excludedRanges = new U8Pointer[excludedRangeList.size()][2];
+		excludedRanges = new U8Pointer[excludedRangeList.size()][];
 		excludedRangeList.toArray(excludedRanges);
 		currentExcludedRange = 0;
 	}
-	
-	private U8Pointer adjustedToRange(U8Pointer ptr, U8Pointer base, U8Pointer top)
+
+	protected U8Pointer getRealHeapTop(J9ModronThreadLocalHeapPointer threadLocalHeap, U8Pointer heapTop, U8Pointer base, U8Pointer top) throws CorruptDataException
+	{
+		return heapTop;
+	}
+
+	protected U8Pointer getRealHeapAlloc(J9ModronThreadLocalHeapPointer threadLocalHeap, U8Pointer heapAlloc, U8Pointer base, U8Pointer top) throws CorruptDataException
+	{
+		U8Pointer realHeapAlloc = null;
```

Comment:
The default should be `U8Pointer.NULL` (callers are not expecting `null` to be returned).